src/exportable/styles.css

```
@@ -0,0 +1,99 @@
+/*
+ * Self-contained styles for @flexprice/flexprice-ui.
+ *
+ * Consumers import this once: `import '@flexprice/flexprice-ui/style.css'`.
+ * It ships the theme tokens (light + dark) the components rely on, plus only the Tailwind
+ * utilities actually used by the pricing components (scoped via tailwind.flexprice-ui.config.js).
+ *
+ * Theme it by overriding these CSS variables on the `.flexprice-ui` element (values are HSL
+ * channels, matching the tokens defined below), e.g.:
+ *   .flexprice-ui { --primary: 243 75% 59%; --radius: 12px; }
+ */
+@tailwind base;
```

Stylelint (17.14.1)

[error] Unexpected unknown at-rule "`@tailwind`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)